### PR TITLE
update routes and currency name

### PR DESCRIPTION
**Pull request by ngvtuan**
##### Updates/Fixes
- [x] Update the new routes from json to ts
- [x] Update all components/ whole app to get the currency name from the appInfo
- [x] Add links for button/link/menu
- [x] Add hover state for button/link/menu/icons
- [x] Fix storybook issues
- [x] Add links for NFTs, Profile screen
- [x] Fix slider on the My security screen

##### FreedCamp
- [x] [update the code for routes and currency name](https://freedcamp.com/view/2956183/tasks/41108913)

### CODE REVIEW

File: src/features/nft/addNFT/fullScreenButton/FullScreenButton.tsx [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
 import { Expand } from 'common/components/Icons'
-import style from './FullScreenButton.module.css'
 
 type TProps = {
   onClick(): void
+  disabled?: boolean
 }
 
-export default function FullScreenButton({ onClick }: TProps): JSX.Element {
+export default function FullScreenButton({
```
Comment (sanjanadhikari1) [inline src/features/nft/addNFT/fullScreenButton/FullScreenButton.tsx:10]: I think it's better to use arrow function for keeping the same style throughout the project.
const FullScreenButton = () => {}

File: src/features/nft/portfolio/PortfolioPage.tsx [hunk 2]
```diff
@@ -19,49 +20,47 @@ import Modal6 from 'features/nft/nftModals/CopiesDetailsModal'
...
-    'There is something magical about space and astronauts. Who wouldn’t want to see the earth from space, or run around on the moon? Space is the final frontier for mankind and essentialy to the future of our species. This piece is an hommage to the brave men and women who represent mankind in this most ambitious of endeavors, who strive to push',
-}
-
 const breadcrumbs = [
   { title: 'My NFT Portfolio', link: '#' },
   { title: '“Super Nfty Floating Head”' },
 ]
 
-nft.description += nft.description // make it longer for "read more"
+export default function PortfolioPage(): JSX.Element {
```
Comment (sanjanadhikari1) [inline src/features/nft/portfolio/PortfolioPage.tsx:28]: Can we use Arrow function for this component?

**Review by sanjanadhikari1 — CHANGES_REQUESTED** [review]

File: src/features/nft/addNFT/fullScreenButton/FullScreenButton.tsx (hunk 1 above)
Comment (CrazyOptimist) [reply src/features/nft/addNFT/fullScreenButton/FullScreenButton.tsx:10]: It's more compact to use function with default export
And with `const` you need to scroll down the file to see if it is default exported

```js
const FullScreenButton = () => {}

export default FullScreenButton
```
vs
```js
export default function FullScreenButton() {}
```

File: tailwind.config.js [hunk 3]
```diff
@@ -410,7 +435,7 @@ module.exports = {
         '15px': '0.938rem',
         '18px': '1.125rem',
         '19px': '1.1875',
-        '30px': '2rem',
+        '30px': '1.875',
```
Comment (CrazyOptimist) [inline tailwind.config.js:438]: No unit specified here, and now we can use `px` as unit

**Review by CrazyOptimist — APPROVED** [review]

File: tailwind.config.js (hunk 3 above)
Comment (CrazyOptimist) [inline tailwind.config.js:438]: No unit specified here, and now we can use px as unit
Some styles may be broken by this, so can't approve

**Review by CrazyOptimist — CHANGES_REQUESTED** [review]

File: src/features/nft/portfolio/PortfolioPage.tsx (hunk 2 above)
Comment (ngvtuan) [reply src/features/nft/portfolio/PortfolioPage.tsx:28]: @sanjanadhikari1 please take a look at this comment https://github.com/pastelnetwork/pastel-electron-wallet/pull/186#discussion_r674171369

File: src/features/nft/addNFT/fullScreenButton/FullScreenButton.tsx (hunk 1 above)
Comment (ngvtuan) [reply src/features/nft/addNFT/fullScreenButton/FullScreenButton.tsx:10]: agreed with @CrazyOptimist

File: tailwind.config.js (hunk 3 above)
Comment (ngvtuan) [reply tailwind.config.js]: Updated

File: src/common/routes/Routes.tsx [hunk 4]
```diff
@@ -152,6 +138,7 @@ const Routes: React.FC = () => {
           )}
         />
       </Switch>
+      <Utilities />
```
[the remark this answers is not in the document]
Comment (ngvtuan) [reply src/common/routes/Routes.tsx]: Updated

**Review by CrazyOptimist — APPROVED** [review]

**Review by sanjanadhikari1 — APPROVED** [review]

**Pull request merged**